example.php

```diff
@@ -19,10 +19,22 @@
 
 /**
  * Include the TGM_Plugin_Activation class.
+ *
+ * Depending on your implementation, you may want to change the include call:
+ *
+ * Parent Theme:
+ * require_once get_template_directory() . '/path/to/class-tgm-plugin-activation.php';
```

Comment:
No need for this - parent themes can call `get_stylesheet_directory()` too, since template dir = stylesheet dir in parent themes.